docs_outdoornav_user_manual/integration_requirements/installation_instructions.mdx

```diff
@@ -0,0 +1,133 @@
+
+``` bash
+cd ~/cpr_outdoornav_launch/scripts/
+./configure_outdoornav.sh
+```
+
+### Finalize Setup {#final-setup}
+
+The script in the sections below will reboot the computer it is run on.
+
+##### Platform Computer
+
+For installations of the OutdoorNav software on the platform computer (not a secondary/backpack/Starter Kit computer), run the following:
+
+``` bash
+cd ~/cpr_outdoornav_launch/
+pushd scripts
+sudo ./setup_computers.sh
+```
+##### Secondary/Starter Kit Computer
+
+Prior to running the script on the secondary/starter kit computer, ensure that you have the user and IP of the platform computer that the secondary/starter kit computer is connected to. Run the following:
+
+``` bash
+cd ~/cpr_outdoornav_launch/
+pushd scripts
+sudo ./setup_computers.sh -b
+```
+
+### Install OutdoorNav Software {#install-outdoornav}
+
+``` bash
+cd ~/cpr_outdoornav_launch/scripts/
+docker compose --profile outdoornav pull
+```
+
+:::note
+If you are installing the OutdoorNav software on a secondary/backpack/Starter Kit computer, you must also complete the [Platform Computer Checklist](/docs_outdoornav_user_manual/integration_requirements/platform_computer_checklist)
+:::
+
+### Configure Platform Footprint {#configure-platform-footprint}
+
+If the platform has sensors and/or parts that protrude outside of the platform body, then a few environment variables will need to be modified to adjust the footprint of the robot. Things that could be extending past the footprint culd include but not limited to:
+
+-   GPS antennae,
+-   Charging receivers,
+-   Arms,
+-   etc...
+
+Change the environment variables fmor the Table below, in the following file:
+
+```
+cd ~/cpr_outdoornav_launch
+nano outdoornav_tuning.env
+```
+
+| Environment Variable        | Description                            | Default |
+|-----------------------------|----------------------------------------|---------|
+| **FOOTPRINT_OFFSET_POS_X**  | Distance from base_link to the furthest edge of any part/sensor at the front of the robot | 0.5   |
+| **FOOTPRINT_OFFSET_NEG_X**  | Distance from base_link to the furthest edge of any part/sensor at the rear of the robot  | -0.5  |
+| **FOOTPRINT_OFFSET_POS_Y**  | Distance from base_link to the furthest edge of any part/sensor at the left of the robot  | 0.35  |
+| **FOOTPRINT_OFFSET_NEG_Y**  | Distance from base_link to the furthest edge of any part/sensor at the right of the robot | -0.35 |
+
+
+
+### Starting OutdoorNav
+
+``` bash
+cd ~/cpr_outdoornav_launch/    # directory with docker-compose.yaml
+docker compose --profile outdoornav up -d
+```
+
+### Testing OutdoorNav Installation
+
+1. Ping all of the network sensors to ensure proper communication with the platform/secondary/backpack computer.

```

Comment:
"platform/secondary/backpack computer" -> "either the platform or secondary computer"